dimagi/utils/django/cached_object.py

```diff
@@ -176,27 +176,9 @@ def __init__(self, cache_key):
         """
         self.cache_key = cache_key
 
-        #key_hash is the hash of the key for eventual use as a retrieval via static assets.
-        #todo: once key_hash determined with object stream, create stream file and deposit onto static machine
-        #generate url and retrieve by key_hash url.
-        self.key_hash = hashlib.md5(cache_key).hexdigest()
-        try:
-            stream_keys, meta_keys = self.get_all_keys()
-            self.stream_keys = stream_keys
-            self.meta_keys = meta_keys
-        except AssertionError as e:
-            logging.exception(e.message)
-
     def is_cached(self):
-        try:
-            metas, streams = self.get_all_keys()
-            if len(metas) == 0:
-                return False
-            else:
-                return True
-        except AssertionError:
-            # maybe one of our keys was purged. Oh well.
-            return False
+        return self.rcache.exists(self.stream_key(OBJECT_ORIGINAL)) and self.rcache.exists(
```

Comment:
@benrudolph  Just realized that this requires an upgrade to the redis client library here: https://github.com/dimagi/commcare-hq/blob/master/requirements/requirements.txt#L64

Might also want to look at upgrading django-redis